Lab 2.ipynb

```diff
@@ -0,0 +1,644 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$\\text{Michael Higgins, Sierra Wilde}$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# $$\\text{Lab 2: Probability, Distributions, and Statistical Questions}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 1:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A)."
```

Comment:
Based on the comments we received yesterday, I think we might need to explain what the Poisson distribution is in the write-up